src/main/Video.tsx

```diff
@@ -58,8 +59,12 @@ const Video: React.FC<{}> = () => {
       } else {
         dispatch(setError({error: true, errorTitle: undefined, errorMessage: t("video.comError-text"), errorDetails: error, errorIcon: undefined}))
       }
+    } else if (videoURLStatus === 'success') {
+      if (!duration) {
```

Comment:
This branch will also be taken for `duration === 0`. While yes, videos with such a duration are not useful anyway, this should likely be `if (duration == null)`. However, TypeScript will complain about that as the type of `duration` is `number` and not `number | null`. This should be fixed (in `videoSlice.ts` I think) so that the type system reflects reality.